docker-compose.yml

```diff
@@ -1,5 +1,10 @@
 version: "3"
 
+x-database: &postgres-credentials
```

Comment:
This is a set of variables. As you can see below, these variables can now be used in multiple locations which makes changing these values very easy.